login-workflow/src/screens/EulaScreen/EulaScreen.tsx

```diff
@@ -93,6 +97,12 @@ export const EulaScreen: React.FC<EulaScreenProps> = (props) => {
         void loadAndCacheEula();
     }, [loadAndCacheEula]);
 
+    const {
+        onRefetch = (): void => {
+            void loadAndCacheEula();
+        },
+    } = props;
```

Comment:
It would probably be a good idea to clear the error immediately upon refetch instead of in the try block. (remember to use whichever new variable name you choose)
```suggestion
    const {
        onRefetch = (): void => {
        setIsError(false)
            void loadAndCacheEula();
        },
    } = props;
```